src/components/HeaderForm.js

```diff
@@ -825,6 +832,19 @@ class HeaderForm extends Component {
     this.setState({ simplify: !this.state.simplify });
   };
 
+  /* Function for toggling nodeSequences button, enabling client to make request for node sequences to be displayed or note */
+  toggleIncludeSequences = () => {
+    this.setState({ removeSequences: !this.state.removeSequences });
+  };
+
+  openPopup = () => {
+    this.setState({ popupOpen: !this.state.popupOpen });
+  };
+
+  closePopup = () => {
+    this.setState({ popupOpen: !this.state.popupOpen });
+  };
```

Comment:
These are called `openPopup` and `closePopup` but really they are two copies of a toggle-the-popup function. Either they should always do what their names say, or there should just be one `togglePopup`.